### ISSUE TITLE
The Visualizer doesn't support a list of bounding_boxes when providing a list of data. 

### ISSUE THREAD

**Issue by tchaton** [isl-org/Open3D-ML#319]
Dea Open3d-ML Team,

The following configuration will break.

```py
data = [data_1, data_2]
bounding_boxes = [bounding_box_1, bounding_box_2]
viz.visualize(data, bounding_boxes=bounding_boxes)
```

Best,
T.C

**Comment by ArrogantNobody** [comment]
Hi:
I also have some issues regarding visualizing bounding boxes, is bounding_box_1 a list that contains coordinates?

**Comment by sanskar107** [comment]
This is fixed in #392
